CRM/Utils/Check/Component/Env.php

```diff
@@ -835,6 +835,25 @@ public function checkDbEngine() {
     return $messages;
   }
 
+  /**
+   * ensure reply id is set to any default value
+   * @return array
+   */
+  public function checkReplyIdForMailing() {
+    $messages = array();
+
+    if (!CRM_Mailing_PseudoConstant::defaultComponent('Reply', '')) {
+      $messages[] = new CRM_Utils_Check_Message(
+        __FUNCTION__,
+        ts('Reply Auto Responder is not set to any default value in <a href=%1>Headers, Footers, and Automated Messages</a>. This will disable the submit operation on any mailing created from CiviMail.', array(1 => CRM_Utils_System::url('civicrm/admin/component', 'reset=1'))),
```

Comment:
I think this html will be malformed because of lack of quotes around the value of href. You could stick the quotes in and that would fix it, but while we're at it, my personal preference is to supply all the attributes of the tag in the argument, so that the tag simply reads `<a %1>` and the 2nd param to ts would be `[1 => 'href="' . CRM_Utils_System::url('civicrm/admin/component', 'reset=1') . '"']`. The reason I prefer this syntax is that the translatable string remains the same even if we add other attributes to the html such as `'target="_blank"'`. No need to re-translate it into 50 languages just due to html changes.

Reply:
Updated the PR to use the anchor tag as mentioned above. Thanks @colemanw 👍